include/sys/vm_physmem.h

```diff
@@ -16,6 +16,9 @@ void _vm_physseg_plug(paddr_t start, paddr_t end, bool used);
 /* Allocates contiguous big page that consists of n machine pages. */
 vm_page_t *vm_page_alloc(size_t n);
 
+/* Allocates pages as vm_page_alloc and zreos them. */
+vm_page_t *vm_page_alloc_zero(int npages);
```

Comment:
`size_t n` as in function above?

Reply:
I have changed all uses of `int` to `size_t`.

Reply:
Have you actually missed this one?

Reply:
Right. Fixed.